src/okta_inspector/reporters/irap.py

```
@@ -0,0 +1,365 @@
+            for vid, _ in area_controls
+            if vid in findings_by_id
+        )
+
+        total_controls = sum(len(v) for v in _ISM_CONTROLS.values())
+
+        lines: list[str] = [
+            "# IRAP Compliance Report - ISM Control Assessment",
+            "",
+            f"**Audit Timestamp:** {ts}  ",
+            f"**Okta Domain:** {data.domain}  ",
+            f"**Assessment Basis:** Australian Government Information Security Manual (ISM)",
```

Ruff (0.15.6)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)